src/CalcViewModel/UnitConverterViewModel.cpp

```diff
@@ -473,14 +504,12 @@ void UnitConverterViewModel::OnButtonPressed(Platform::Object ^ parameter)
     }
 
     static constexpr UCM::Command OPERANDS[] = { UCM::Command::Zero, UCM::Command::One, UCM::Command::Two,   UCM::Command::Three, UCM::Command::Four,
-                                                   UCM::Command::Five, UCM::Command::Six, UCM::Command::Seven, UCM::Command::Eight, UCM::Command::Nine };
-    if (m_isInputBlocked &&
-        command != UCM::Command::Clear &&
-        command != UCM::Command::Backspace)
-	{
-		return;
-	}
-	m_model->SendCommand(command);
+                                                 UCM::Command::Five, UCM::Command::Six, UCM::Command::Seven, UCM::Command::Eight, UCM::Command::Nine };
+    if (m_isInputBlocked && !m_model->IsSwitchedActive() && command != UCM::Command::Clear && command != UCM::Command::Backspace)
+    {
+        return;
+    }
+    m_model->SendCommand(command);
```

Comment:
`!m_model->IsSwitchedActive()` means input is allowed as long as user just switches active.

Reply:
I've added the same comment in code.